src/qibotn/backends/quimb.py

```diff
@@ -24,12 +24,40 @@ def __init__(self, runcard):
             else:
                 raise TypeError("MPS_enabled has an unexpected type")
 
+            global TEBD_enabled
+            global TEBD_option
+            TEBD_enabled = runcard.get("TEBD_enabled")
+            tebd_enabled_value = runcard.get("TEBD_enabled")
```

Comment:
Why do you have two identical variables?

I'm pretty sure you can throw away one between `TEBD_enabled` and `tebd_enabled_value`, just replacing the occurrences other than definition with the variable name you decide to keep.